code/modules/events/blob.dm

```diff
@@ -28,7 +28,6 @@
 	var/mob/M = pick(candidates)
 	B.key = M.key
 	B.mind.special_role = SPECIAL_ROLE_BLOB
-	SSticker.mode.update_blob_icons_added(B.mind)
```

Comment:
won't this mean that blob mice won't get antag huds? not a major issue as it calls them out to all ghosts.

Reply:
Blob mice having the blob antag icon is a QoL that I really don't want to see going away, would be good to keep it.

Reply:
Echoing what's been said here, this is something that extends beyond round start blobs so it seems like an extraneous change for this pr

Reply:
Re-added